EduVPN/AppDelegate.swift

```diff
@@ -85,6 +85,15 @@ class AppDelegate: NSObject, NSApplicationDelegate {
             NSApp.activate(ignoringOtherApps: true)
         }
 
+        NSWorkspace.shared.notificationCenter.addObserver(
+            forName: NSWorkspace.didWakeNotification, object: nil,
```

Comment:
See linting error. Since you probably will never want to un subscribe, silence it with `_ =`?

Reply:
I didn't get a SwiftLint error. Is this what you had in mind: commit dab2bae ?